test/parallel/test-zlib-write-after-close.js

```diff
@@ -29,9 +29,9 @@ zlib.gzip('hello', common.mustCall(function(err, out) {
   common.expectsError(
     () => unzip.write(out),
     {
-      code: 'ERR_ZLIB_BINDING_CLOSED',
```

Comment:
Can this error still be triggered at all? I don’t see how, so it’s probably best to remove it from the errors list, from the docs, and turn the handle check in `lib/zlib.js` into an assertion.